buildSrc/src/main/java/net/neoforged/neodev/GenerateSourcePatches.java

```diff
@@ -25,9 +27,14 @@ abstract class GenerateSourcePatches extends DefaultTask {
     @PathSensitive(PathSensitivity.RELATIVE)
     public abstract DirectoryProperty getModifiedSources();
 
+    @Optional
     @OutputFile
     public abstract RegularFileProperty getPatchesJar();
 
+    @Optional
+    @OutputDirectory
+    public abstract DirectoryProperty getPatchesFolder();
```

Comment:
I don't think that's necessary. We can write the userdev patches to a jar, no?

Reply:
We need to unpack them to a folder if we want to give them to binpatcher